moodlecli/main.py

```diff
@@ -250,3 +250,33 @@ def export_users_by_course(source_course_id, bucket_name, key):
     )
 
     aws.put_json_data(users, bucket_name, key)
+
+
+@cli.command()
+@click.argument('bucket_name')
+@click.argument('directory')
+def export_users_bulk(bucket_name, directory):
+    "Output the users for a series of classes to an s3 bucket"
+    moodle = get_moodle_client()
+
+    course_data = moodle.get_courses()
```

Comment:
A problem with this approach is it assumes all courses on the instance are relevant / desired. Given our S3 convention is semester based, and we probably don't want to pull / rewrite data for old courses anyways, I think it makes more sense for this and the other bulk export command to take a CSV file as input. In this case it only needs a single column of `course_id`. It may be nice to include a helper `export-bulk-csv` command that we can use to create the empty input CSV as well.